### ISSUE TITLE
Mod gear parts completely disappear in game 

### ISSUE THREAD

**Issue by paikomango** [xivdev/Penumbra#302]
It's my first time use this plugin and I did few mods in game, hair or face works fine but the gear is completely went invisible in game. Can someone help me with this issue?

**Comment by Theo-Asterio** [comment]
Yes, but the more ideal place to handle this would be on the discord, you haven't provided enough information. More than likely whatever gear you have requires a body and you need to go read the mod page for it, github isn't for mod support, penumbra discord can help better.
